deploy/olm-catalog/pulp-operator/manifests/pulp-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,295 @@
+        - urn:alm:descriptor:com.tectonic.ui:fieldDependency:ingress_type:Ingress
+      - displayName: Pulp Ingress TLS Secret
+        path: ingress_tls_secret
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:advanced
+        - urn:alm:descriptor:io.kubernetes:Secret
+        - urn:alm:descriptor:com.tectonic.ui:fieldDependency:ingress_type:Ingress
+      - displayName: Route DNS host
+        path: route_host
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:advanced
+        - urn:alm:descriptor:com.tectonic.ui:text
+        - urn:alm:descriptor:com.tectonic.ui:fieldDependency:ingress_type:Route
+      - displayName: Route TLS termination mechanism
+        path: route_tls_termination_mechanism
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:advanced
+        - urn:alm:descriptor:com.tectonic.ui:select:Edge
+        - urn:alm:descriptor:com.tectonic.ui:select:Passthrough
+        - urn:alm:descriptor:com.tectonic.ui:fieldDependency:ingress_type:Route
+      - displayName: Route TLS credential secret
+        path: route_tls_secret
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:advanced
+        - urn:alm:descriptor:io.kubernetes:Secret
+        - urn:alm:descriptor:com.tectonic.ui:fieldDependency:ingress_type:Route
+      - displayName: Deployment Type
+        path: deployment_type
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:advanced
+        - urn:alm:descriptor:com.tectonic.ui:hidden
+      - displayName: Pulp Image
+        path: image
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:advanced
+        - urn:alm:descriptor:com.tectonic.ui:hidden
+      - displayName: Postgres Storage Class
+        path: postgres_storage_class
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:advanced
+        - urn:alm:descriptor:com.tectonic.ui:hidden
+      - displayName: Certificate Authorirty Trust Bundle
+        path: ca_trust_bundle
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:advanced
+        - urn:alm:descriptor:com.tectonic.ui:hidden
+    statusDescriptors:
+      - displayName: URL
+        description: Route to access the instance deployed
+        path: webURL
+        x-descriptors:
+          - urn:alm:descriptor:org.w3:link
+      - displayName: Admin Password
+        description: Admin password for the instance deployed
+        path: sdminPasswordSecret
+        x-descriptors:
+          - urn:alm:descriptor:io.kubernetes:Secret
+      - displayName: Version
+        description: Version of the instance deployed
+        path: deployedVersion
+        x-descriptors:
+          - urn:alm:descriptor:com.tectonic.ui:text
+      - displayName: Image
+        description: Image of the instance deployed
+        path: deployedImage
+        x-descriptors:
+          - urn:alm:descriptor:com.tectonic.ui:text
+      version: v1beta1
```

Comment:
This should be moved to line 36, along side `kind` and `name`

Reply:
Will fix. side effect of copy/paste from awx-operator: https://github.com/ansible/awx-operator/blob/devel/deploy/olm-catalog/awx-operator/manifests/awx-operator.clusterserviceversion.yaml#L231